### PR TITLE
Added new Slack workspace to README

### PR DESCRIPTION
**Pull request by DragonRaider5**
Hey,

to coordinate the work on the PR to use the new context API, I created a Slack workspace for this package and I think it might as well serve other purposes, such as help and an overall better community experience.

Would be happy to see this merged as soon as possible and I hope you aren't upset about me creating a workspace using this name. If you want to I can make you the owner/admin ASAP.

Best Regards,
DragonRaider5

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by StorytellerCZ** [comment]
How can you join that workspace?

**Comment by DragonRaider5** [comment]
@StorytellerCZ you can simply take the link ([react-intl.slack.com](https://react-intl.slack.com)) and register there. Alternatively you can join the workspace `react-intl.slack.com` using the Slack client.

**Comment by StorytellerCZ** [comment]
@DragonRaider5 Actually I need an invitation:
```
Don't have an account on this workspace yet?
Contact the workspace administrator for an invitation 
```
That is the problem with Slack spaces. They are inherently a closed spaces. Though I think there is a Heroku app that you can deploy to automatically send invites to people who request them.

**Comment by DragonRaider5** [comment]
Haha my bad - I didn't even know this. I just knew the principle from the `parcel-bundler` project, where they seem to use (and host) slackin for this purpose...

Maybe we should come up with an alternative to Slack? I found Flock, other ideas?

**Comment by StorytellerCZ** [comment]
[Gitter](https://gitter.im) is tight pretty closely to Github so that might work.

**Comment by DragonRaider5** [comment]
Thanks to @papasmile pointing me at the documentation, I now obtained a general invite link.
I'll update it in the README - then it should work!

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ Overview
 
 **React Intl is part of [FormatJS][].** It provides bindings to React via its components and API.
 
+**New:** Join us on Slack at [react-intl.slack.com](https://react-intl.slack.com/) for help, general conversation and more 💬🎊🎉
```
Comment (redonkulus) [inline README.md]: nit: instead of saying "New", can it say "Slack" just because eventually it won't be New anymore.

**Pull request merged**